services/controller/src/plz/controller/results/local.py

```diff
@@ -13,15 +15,19 @@
 
 
 class LocalResultsStorage(ResultsStorage):
-    def __init__(self, redis: StrictRedis, directory: str):
+    def __init__(self, redis: StrictRedis, db_storage: DBStorage,
+                 directory: str):
```

Comment:
If you're gonna put something on a new line, put each argument on a new line please.

Reply:
We weren't following this. That's Google style so not extraneous to me, but blind usage of the rule can end up with things like
`func(
    somethingVeryLong,
    1,
    false,
    ...a zillion things that would fit in a line and look horrible each in a single line... 
)`
So I think there's some subjectivity in it (or maybe you only want to apply to definitions?). This case doesn't like too bad, so I'm following your advise.